msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -720,3 +720,20 @@ You can define any of the following properties:
 * `developerUrl`: The HTTPS URL of the developer's website.
 * `privacyUrl`: The HTTPS URL of the developer's privacy policy.
 * `termsOfUseUrl`: The HTTPS URL of the developer's terms of use.
+
+## defaultBlockUntilAdminAction
+
+When `defaultBlockUntilAdminAction` property is set to true, the app is hidden from users by default until Admin allows it.
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`type`|boolean|32 characters|✔|If set to **true**, the app is hidden for all tenants and end users. The tenant admins can see the app in the Teams admin center and take action to allow or block the app. The default value is **false**.|
+|`description`|string|128 characters||The description that appears to users to indicate the purpose of this property.|
+|`default`|||✔|Default is **false**.|
+
+## publisherDocsUrl
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`$ref`|||✔|The URL to the page that provides additional app information for the admins.|
```

Comment:
Publish URL is a string value. This should be document similar to this: https://review.docs.microsoft.com/en-us/microsoftteams/platform/resources/schema/manifest-schema?branch=pr-en-us-4149#packagename